bin/pytorch_inference/Main.cc

```diff
@@ -4,104 +4,117 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+#include <api/CIoManager.h>
```

Comment:
We usually put our headers in the dependency order of the libraries, so core first, api last.  clang-format has changed this because you deleted the blank lines in between.